roles/httpd/tasks/main.yml

```diff
@@ -1,22 +1,9 @@
 ---
-- name: Install Apache httpd
-  ansible.builtin.package:
-    name:
-      - httpd
-      - mod_ssl
+- name: Pull Apache container image
+  containers.podman.podman_image:
+    name: registry.access.redhat.com/ubi9/httpd-24:latest
```

Comment:
Given this is used in two places, a variable in `defaults.yml` would be helpful.